tests/softwareupdate/test_cloud_config.py

```diff
@@ -0,0 +1,668 @@
+            "name": "OctoPrint-Netconnectd Plugin",
+            "repo": "OctoPrint-Netconnectd",
+            "pip": "https://github.com/mrbeam/OctoPrint-Netconnectd/archive/{target_version}.zip",
+            "restart": "octoprint",
+            "dependencies": {
+                "netconnectd-daemon": {
+                    "name": "Netconnectd Daemon",
+                    "repo": "netconnectd_mrbeam",
+                    "pip": "https://github.com/mrbeam/netconnectd_mrbeam/archive/{target_version}.zip",
+                    "global_pip_command": True,
+                    "package_name": "netconnectd",
+                    "beamos_date": {
+                        "2018-01-12": {
+                            "stable": {
+                                "branch": "mrbeam2-stable",
+                                "branch_default": "mrbeam2-stable",
+                            },
+                            "beta": {
+                                "branch": "mrbeam2-stable",
+                                "branch_default": "mrbeam2-stable",
+                            },
+                            "develop": {
+                                "branch": "mrbeam2-stable",
+                                "branch_default": "mrbeam2-stable",
+                            },
+                            "alpha": {
+                                "branch": "mrbeam2-stable",
+                                "branch_default": "mrbeam2-stable",
+                            },
+                        },
+                        "2021-06-11": {
+                            "pip_command": "sudo /usr/local/netconnectd/venv/bin/pip",
+                            "stable": {"branch": "master", "branch_default": "master"},
+                        },
+                    },
+                }
+            },
+        },
+        "findmymrbeam": {
+            "name": "OctoPrint-FindMyMrBeam",
+            "repo": "OctoPrint-FindMyMrBeam",
+            "pip": "https://github.com/mrbeam/OctoPrint-FindMyMrBeam/archive/{target_version}.zip",
+            "restart": "octoprint",
+        },
+    },
+}
+
+
+@ddt
+class SettingsTestCase(unittest.TestCase):
+    _softwareupdate_handler = None
+    plugin = None
+
+    def setUp(self):
+        self.plugin = MrBeamPluginDummy()
+
+    def test_server_not_reachable(self):
+        plugin = self.plugin
+        tier = "DEV"
+        beamos_date = date(2018, 1, 12)
+        with requests_mock.Mocker() as rm:
+            rm.get(
+                "https://api.github.com/repos/mrbeam/beamos_config/tags",
+                json={"test": "test"},
+                status_code=404,
+            )
+            rm.get(
+                "https://api.github.com/repos/mrbeam/beamos_config/contents/docs/sw-update-conf.json?ref=vNone",
+                status_code=404,
+            )
+            tag_of_github_repo = get_tag_of_github_repo("beamos_config")
+            assert tag_of_github_repo == None
```

Comment:
since you are on a TestCase you could use self.assertIsNone https://docs.python.org/3/library/unittest.html#assert-methods 😄